src/Servers/HttpSys/src/NativeInterop/UrlGroup.cs

```diff
@@ -40,24 +40,6 @@ internal unsafe UrlGroup(ServerSession serverSession, ILogger logger)
         Id = urlGroupId;
     }
 
-    internal unsafe UrlGroup(RequestQueue requestQueue, UrlPrefix url, ILogger logger)
-    {
-        _logger = logger;
-
-        ulong urlGroupId = 0;
-        _created = false;
-        var statusCode = HttpApi.HttpFindUrlGroupId(
```

Comment:
One side effect of this change is when an invalid prefix is given. Before this change, it would fail during rule creation, now it fails later when trying to delegate the request. I think this is more desirable as there is more flexibility to create the rules before the groups exist.